ReactiveExtensions/operators/WithLatestFrom.swift

```diff
@@ -10,7 +10,7 @@ public extension SignalProtocol {
 
    - returns: A new signal.
    */
-  public func withLatestFrom <U, OtherError: Swift.Error> (_ other: Signal<U, OtherError>) ->
```

Comment:
Redundant conformity (`Signal`'s `Error` associated type must conform to `Swift.Error` already).